src/controller/python/chip/clusters/Attribute.py

```diff
@@ -401,68 +401,73 @@ def UpdateCachedData(self):
         tlvCache = self.attributeTLVCache
         attributeCache = self.attributeCache
 
-        for endpoint in tlvCache:
+        for attributePath in changedPathSet:
+            endpoint = attributePath.EndpointId
+
             if (endpoint not in attributeCache):
                 attributeCache[endpoint] = {}
 
             endpointCache = attributeCache[endpoint]
 
-            for cluster in tlvCache[endpoint]:
-                if cluster not in _ClusterIndex:
+            cluster = attributePath.ClusterId
+
+            if cluster not in _ClusterIndex:
+                #
+                # #22599 tracks dealing with unknown clusters more
+                # gracefully so that clients can still access this data.
```

Comment:
Would be great to fix that soon :)

Reply:
Well, we actually thinking about ways of getting rid of the cache, as we don't really need it on server side... I guess others still need it, so we'd make it an opt-in thing or something.